src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -10680,7 +10680,7 @@ def test_aks_update_with_azuremonitormetrics(
 
     #     # update: disable-azure-container-storage
     #     update_cmd = 'aks update --resource-group={resource_group} --name={name} --yes --output=json ' \
-    #                  '--disable-azure-container-storage'
+    #                  '--disable-azure-container-storage all'
```

Comment:
> Introduce this test back once v1.0.3-preview version of Azure container storage is released
This case is still commented out, is it the time to add the case back?
Queue [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=89272608&view=results) to validate the change.
- test_aks_create_with_azurecontainerstorage

Reply:
We need to wait for version `1.0.6-preview` since these changes need the latest version to get validated.

Reply:
This test is now fixed and enabled.